ccd-definition/CaseEventToFields/CareSupervision/manageOrders/5_orderDetailsSection.json

```diff
@@ -29,12 +29,130 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "manageOrders",
-    "CaseFieldID": "manageOrdersFurtherDirections",
+    "CaseFieldID": "manageOrdersEpoType",
     "PageFieldDisplayOrder": 3,
+    "DisplayContext": "MANDATORY",
+    "PageID": "OrderDetails",
+    "PageDisplayOrder": 5,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "orderTempQuestions.epoOrderType=\"YES\"",
```

Comment:
I feel like this could have a more generic name, something like just `orderTempQuestions.epo` or something like that so that you don't need to specifiy a new field for each question to be dislpayed when it is an epo.
This field would then replace `epoOrderType`, `epoPreventRemoval`, `epoExpiryDate`, `epoIncludePhrase` and `epoChildrenDescription`